code/modules/jobs/job_exp.dm

```diff
@@ -312,15 +320,15 @@ GLOBAL_PROTECT(exp_to_update)
 				"job" = "'[sanitizeSQL(jtype)]'",
 				"ckey" = "'[sanitizeSQL(ckey)]'",
 				"minutes" = play_records[jtype])))
-	addtimer(CALLBACK(GLOBAL_PROC,.proc/update_exp_db),20,TIMER_OVERRIDE|TIMER_UNIQUE)
+	addtimer(CALLBACK(SSblackbox,/datum/controller/subsystem/blackbox.proc/update_exp_db),20,TIMER_OVERRIDE|TIMER_UNIQUE)
```

Comment:
Convention says you should put 

/datum/controller/subsystem/blackbox/proc/update_exp_db

instead of 

/datum/controller/subsystem/blackbox.proc/update_exp_db